subprojects/plugins/src/main/java/org/gradle/api/plugins/internal/DefaultJavaPluginExtension.java

```diff
@@ -104,49 +207,38 @@ public void registerFeature(String name, Action<? super FeatureSpec> configureAc
         spec.create();
     }
 
-    @Override
-    public void disableAutoTargetJvm() {
-        convention.disableAutoTargetJvm();
-    }
-
-    @Override
     public void withJavadocJar() {
         TaskContainer tasks = project.getTasks();
         ConfigurationContainer configurations = project.getConfigurations();
-        SourceSet main = convention.getSourceSets().getByName(SourceSet.MAIN_SOURCE_SET_NAME);
+        SourceSet main = getSourceSets().getByName(SourceSet.MAIN_SOURCE_SET_NAME);
         configureDocumentationVariantWithArtifact(JAVADOC_ELEMENTS_CONFIGURATION_NAME, null, JAVADOC, ImmutableList.of(), main.getJavadocJarTaskName(), tasks.named(main.getJavadocTaskName()), findJavaComponent(components), configurations, tasks, objectFactory);
     }
 
-    @Override
```

Comment:
Why remove the Override annotations here and below?

Reply:
That's a leftover from the convention -> extension conversion. Fixed it.